changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -63,81 +69,8 @@ Studios is Seqera's in-platform tool for secure, on-demand, interactive data ana
-
-`jdbc:mysql://<domain>:<port>/tower?permitMysqlScheme=true&sessionVariables=sql_mode='STRICT_TRANS_TABLES,NO_ENGINE_SUBSTITUTION'`
-
-### Micronaut property key changes
-
-The property that determines the expiration time of the JWT access token (used for authenticating web sessions and Nextflow-Platform interactions) has changed as of Seqera Enterprise version 24.1:
-
-| Previous                                                         | New                                                          |
-| ---------------------------------------------------------------- | ------------------------------------------------------------ |
-| `micronaut.security.token.jwt.generator.access-token.expiration` | `micronaut.security.token.generator.access-token.expiration` |
-	
-Enterprise deployments that have customized this value previously will need to adopt the new format.
-
-## Upgrade steps 
-
-:::warning
-Upgrading to version 25.1 requires backend downtime in order for the database migration to complete successfully.
-:::
-
-1. This version includes an update to the Platform Enterprise H8 cache. **Do not start the upgrade while any pipelines are in a `running` state as active run data may be lost.**
-1. This version requires a database schema update. Make a backup of your Platform database prior to upgrade.
-1. If you are upgrading from a version older than 23.4.1, update your installation to version [23.4.4](https://docs.seqera.io/changelog/seqera-enterprise/v23.4) first, before updating to 25.1 with the steps below.
-1. For recommended Platform memory settings, add the following environment variable to your Platform configuration values (`tower.env`, `configmap.yml`, etc.):
-    ```bash
-    JAVA_OPTS: -Xms1000M -Xmx2000M -XX:MaxDirectMemorySize=800m -Dio.netty.maxDirectMemory=0 -Djdk.nio.maxCachedBufferSize=262144
-    ```
-1. See [Upgrade installation](https://docs.seqera.io/platform-enterprise/24.2/enterprise/upgrade) for installation upgrade guidance.
 
-:::info
-Docker Compose deployments require downtime while upgrading services. Restarting the application may take several minutes. See [Docker compose deployment](https://docs.seqera.io/platform-enterprise/latest/enterprise/docker-compose) for more information.
+:::note[UPGRADING]
 
-For Kubernetes deployments, apply the 25.1 `tower-cron.yml` to your cron pod and wait for the cron pod to be running before applying the `tower-svc.yml` to your backend pod and restarting the service. If the cron pod update is interrupted, you may need to restore the instance from your DB backup and start again. See [Kubernetes deployment](https://docs.seqera.io/platform-enterprise/latest/enterprise/kubernetes) for more information. 
+See [Upgrade installation](https://docs.seqera.io/platform-enterprise/25.1/enterprise/upgrade) for installation upgrade guidance.
 
-For custom deployments with third-party services such as ArgoCD, [contact support](https://support.seqera.io) for assistance during upgrade. 
-::: 
```

Comment:
Removed breaking changes as these specifically were introduced in older versions of the platform and now live in the breaking-changes file.

We should continue to publish them here as-well when they are relevant.